ocaml/idl/datamodel.ml

```diff
@@ -10346,6 +10404,7 @@ let all_relations =
     (_vusb, "VM"), (_vm, "VUSBs");
 
     (_feature, "host"), (_host, "features");
+    (_network_sriov, "physical_PIF"), (_pif, "sriov_physical_PIF_of");
```

Comment:
What about the one for the logical PIF?

Reply:
After recheck the datamodel definition of  `vlan` and `tunnel`, I think we should also refine `PIF.sriov_logical_PIF_of` with type `Set(Ref _network_sriov)`  as `tunnel` did because it was introduced later than `vlan` so we should treat it as the latest style and keep consistent in the design.

Reply:
Indeed, `vlan` is a bit inconsistent.